motionit/src/main/java/com/back/motionit/global/error/code/CommentErrorCode.java

```diff
@@ -0,0 +1,18 @@
+package com.back.motionit.global.error.code;
+
+import org.springframework.http.HttpStatus;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum CommentErrorCode implements ErrorCode {
+
+	WRONG_ACCESS(HttpStatus.FORBIDDEN, "U-101", "본인이 작성한 댓글이 아닙니다."),
+	COMMENT_NOT_FOUND(HttpStatus.NOT_FOUND, "M-102", "댓글을 찾을 수 없습니다.");
+
```

Comment:
```suggestion
	ROOM_NOT_FOUND(HttpStatus.NOT_FOUND, "C-404", "운동방을 찾을 수 없습니다."),
	USER_NOT_FOUND(HttpStatus.NOT_FOUND, "M-103?", "사용자를 찾을 수 없습니다.");
```
Service 레이어에서 `getReferenceById()` 대신 `findById()`를 사용하도록 변경함에 따라,
  댓글 생성 시 존재하지 않는 운동방이나 사용자에 대한  에러 응답을위해 에러 코드를 추가 예시 제안입니다.